### ISSUE TITLE
Performance testing and telemetry 

### ISSUE THREAD

**Issue by llali** [Microsoft/sqlopsstudio#560]
- [x] Startup time telemetry report
- [x] Create job to get startup time telemetry data 
- [x] Graphs for start up time
- [ ] Adding telemetry data for extension language clients ready
- [ ] Adding telemetry properties to run command during initialization
- [ ] Defining the scenarios for more telemetries 
- [x] Plan for performance tests
- [x] Performance test for startup
- [x] Performance test report
- [x] Test Iterations
- [x] Verifying test for new builds
- [x] Add telemetry for session duration 
- [x] Clean up cache before starting the app

**Comment by kburtram** [comment]
@llali is this task done?  If so, could you please close?

**Comment by kburtram** [comment]
I think we're done with this task, so I'll close it out.  Please open a new issue for the next planned Performance Testing activities.
